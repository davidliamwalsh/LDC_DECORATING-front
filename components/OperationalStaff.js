import { Component } from 'react'

class OperationalStaff extends Component {
  render () {
    return <>
     <div className="c-about__operational-staff">
       <div className="c-about__operational-staff--image" />
       <p className="c-about__operational-staff--text">
       As of 2016, LDC Decorating Contractors directly employ over 50 tradesmen including professional painters and decorators as well as other trades. All our operatives are among the very best in their profession. Led by experienced site supervisors, they all hold the industry standard CSCS card proving competence in both their work and their approach to health and safety.
       </p>
     </div>
    </>
  }
}

export default OperationalStaff