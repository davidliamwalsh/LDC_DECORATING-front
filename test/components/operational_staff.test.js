import OperationalStaff from '../../components/OperationalStaff'

describe('<OperationalStaff />', () => {
  it('it renders correctly', () => {
    const component = shallow(
      <OperationalStaff />
    )

    expect(component).toMatchSnapshot()
  })
})