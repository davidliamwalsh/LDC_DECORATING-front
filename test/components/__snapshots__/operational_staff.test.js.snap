// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OperationalStaff /> it renders correctly 1`] = `ShallowWrapper {}`;
